src/index.js

```diff
@@ -78,15 +78,16 @@ export default class NestedList {
     this.config = config;
 
     /**
-     * This list-style will be used by default
+     * Set the default list style from the config.
      */
-    this.defaultListStyle = 'ordered';
+    this.defaultListStyle = this.config.defaultStyle === 'unordered'  ? 'unordered' : 'ordered';
```

Comment:
please make `unordered` by default

Reply:
Orderd is default in list tool too. That why i have put the same here

Reply:
so let's update that tool too

Reply:
okay